Formula/nmh.rb

```diff
@@ -36,6 +42,9 @@ def install
                           "--with-cyrus-sasl",
                           "--with-tls"
     system "make", "install"
+
+    # Remove shim references on Linux
+    inreplace etc/"nmh/nmh.defaults", Superenv.shims_path, ""
```

Comment:
May be saved in `prefix/"etc"` rather than `etc`.

Also should probably account for slash: `nmh-access-url: @@HOMEBREW_LIBRARY@@/Homebrew/shims/mac/super/curl -L`, either
- `Superenv.shims_path/"curl" ==> "curl"` or 
- `"#{Superenv.shims_path}/" ==> ""`

Reply:
The filename was `mhn.defaults`, not `nhm.defaults` 🤦.